### PR TITLE
IF: Integration tests enabled for instant-finality (Part 1)

### PR DESCRIPTION
**Pull request by heifner**
Adding additional integration tests for instant-finality
- block_log_util_if_test
- block_log_retain_blocks_if_test
- cluster_launcher_if
- distributed-transactions-if-test
- nodeos_forked_chain_if_lr_test
- nodeos_contrl_c_if_test
- nodeos_chainbase_allocation_if_test
- nodeos_producer_watermark_if_lr_test
- nodeos_high_transaction_if_lr_test
- larger_lib_if_test
- light_validation_sync_if_test
- auto_bp_peering_if_test

There are more that need to be added, but this set gives us much more coverage than we currently have. Merging this now, and will add additional tests in a (Part 2).

### CODE REVIEW

File: tests/CMakeLists.txt [hunk 1]
```diff
@@ -69,7 +69,6 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/resource_monitor_plugin_test.py ${CMA
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/light_validation_sync_test.py ${CMAKE_CURRENT_BINARY_DIR}/light_validation_sync_test.py COPYONLY)
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/trace_plugin_test.py ${CMAKE_CURRENT_BINARY_DIR}/trace_plugin_test.py COPYONLY)
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/nested_container_multi_index_test.py ${CMAKE_CURRENT_BINARY_DIR}/nested_container_multi_index_test.py COPYONLY)
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/large-lib-test.py ${CMAKE_CURRENT_BINARY_DIR}/large-lib-test.py COPYONLY)
```
Comment (linh2931) [inline tests/CMakeLists.txt:72]: Why was this removed?

**Review by linh2931 — APPROVED** [review]
Those should be checked in as soon as possible so we don't introduce new bugs while busy writing large amount of new code.

**Review by greg7mdp — APPROVED** [review]

**Comment by ericpassmore** [comment]
Note:start
group: IF
category: TEST
summary: Added tests to cover more scenarios including block_log, forked chain, larger lib, auto peering, and distributed transactions.
Note: end
